src/styles/colors.scss

```diff
@@ -19,3 +19,9 @@ $list-item-hairline: $brand-bg-light;
 
 $max-width: 800px;
 $desktop-side-column-color: #eeeeee;
+
+$link-text-color: $brand-pane-alt-dark;
+
+a {
+    color: $link-text-color;
```

Comment:
Don't see any other links using that color definition.
I'd rather not use `!important` for stuff that isn't getting overridden by mui.